database-plugins/src/test/java/co/cask/hydrator/plugin/db/batch/sink/BatchETLDBTestRun.java

```diff
@@ -362,6 +346,17 @@ public void testUserNamePasswordCombinations() throws Exception {
     etlConfig = new ETLBatchConfig("* * * * *", table, database, transforms);
     appRequest = new AppRequest<>(ETLBATCH_ARTIFACT, etlConfig);
     TestBase.deployApplication(appId, appRequest);
+    // non-null username, null password. Should succeed.
+    Map<String, String> nullPassword = new HashMap<>(baseSourceProps);
+    emptyPassword.put(Properties.DB.USER, "emptyPwdUser");
+    database = new ETLStage("databaseSource", new Plugin("Database", nullPassword));
+    etlConfig = ETLBatchConfig.builder("* * * * *")
+      .setSource(database)
+      .addSink(table)
+      .addConnection(database.getName(), table.getName())
+      .build();
+    appRequest = new AppRequest<>(ETLBATCH_ARTIFACT, etlConfig);
+    TestBase.deployApplication(appId, appRequest);
```

Comment:
for consistency, should we add a test using DBSink as well? Also, not sure why ``deployApplication`` in this class needs to be called with the class name